service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ManageHearingsControllerAboutToSubmitTest.java

```diff
@@ -308,7 +344,7 @@ void shouldVacateHearing(HearingReListOption adjournmentOption) {
         Element<HearingBooking> expectedVacatedHearing = element(
             futureHearingToBeAdjourned.getId(),
             futureHearingToBeAdjourned.getValue().toBuilder()
-                .status(HearingStatus.VACATED)
+                .status(adjournmentOption == RE_LIST_LATER ? VACATED_TO_BE_RE_LISTED : VACATED)
```

Comment:
I'd avoid to add if statements in tests. You can pass the expected value as a test parameter